src/redux/store.js

```diff
@@ -1,11 +1,13 @@
-import { configureStore } from '@reduxjs/toolkit';
+import { configureStore, applyMiddleware } from '@reduxjs/toolkit';
+import thunk from 'redux-thunk';
 import bookReducer from './books/books';
 import categoryReducer from './categories/categories';
 
 /* eslint-disable import/prefer-default-export */
```

Comment:
[OPTIONAL] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.